### ISSUE TITLE
Expected argument to be an absolute path or URI: '<unknown>'

### ISSUE THREAD

**Issue by conartist6** [microsoft/deoptexplorer-vscode#14]
```
[ERR!: 10:37:57.169] line 19975: Expected argument to be an absolute path or URI: '<unknown>'
TypeError: Expected argument to be an absolute path or URI: '<unknown>'
    at t.pathOrUriStringToUri (/Users/conartist6/.vscode/extensions/rbuckton.deoptexplorer-vscode-1.0.2-darwin-arm64/dist/extension.js:5:155602)
    at t.LogProcessor.processScriptSource (/Users/conartist6/.vscode/extensions/rbuckton.deoptexplorer-vscode-1.0.2-darwin-arm64/dist/extension.js:2:182129)
    at u.dispatchLogRow_ (/Users/conartist6/.vscode/extensions/rbuckton.deoptexplorer-vscode-1.0.2-darwin-arm64/dist/extension.js:165:35835)
    at u.processLogLine_ (/Users/conartist6/.vscode/extensions/rbuckton.deoptexplorer-vscode-1.0.2-darwin-arm64/dist/extension.js:165:36292)
    at u.processLogLine (/Users/conartist6/.vscode/extensions/rbuckton.deoptexplorer-vscode-1.0.2-darwin-arm64/dist/extension.js:165:34581)
    at t.LogProcessor.processContent (/Users/conartist6/.vscode/extensions/rbuckton.deoptexplorer-vscode-1.0.2-darwin-arm64/dist/extension.js:2:174391)
    at t.measureAsync (/Users/conartist6/.vscode/extensions/rbuckton.deoptexplorer-vscode-1.0.2-darwin-arm64/dist/extension.js:5:27865)
    at t.LogProcessor.process (/Users/conartist6/.vscode/extensions/rbuckton.deoptexplorer-vscode-1.0.2-darwin-arm64/dist/extension.js:2:174665)
    at /Users/conartist6/.vscode/extensions/rbuckton.deoptexplorer-vscode-1.0.2-darwin-arm64/dist/extension.js:5:42522
```

Line 19975 is: `script-source,168,<unknown>,'__esModule'`

**Comment by conartist6** [comment]
How can I find out what file this is referring to?

**Comment by rbuckton** [comment]
If you open the log file directly into the editor, you can jump to the line that is being processed at line 19975 of the log.

**Comment by conartist6** [comment]
I pasted that line at the bottom of the issue but it doesn't tell me much...

**Comment by rbuckton** [comment]
for a `script-source` event, I try to parse the 2nd argument as an absolute path or fully-qualified URL, but it seems V8 may write some arbitrary non-path-like things to that entry. This should hopefully be fixed once 1.0.3 is published.
